local_costmap:
   global_frame: /map
   robot_base_frame: /base_footprint
   update_frequency: 10.0
   publish_frequency: 2.0
   static_map: false
   rolling_window: true
   width: 4.0
   height: 4.0
   resolution: 0.05
   transform_tolerance: 0.5


   plugins:
    -
      name: footprint_layer
      type: "costmap_2d::FootprintLayer"
    -
      name: obstacle_layer
      type: "costmap_2d::ObstacleLayer"
    -
      name: ball_picker_layer
      type: "costmap_2d::BallPickerLayer"
    -
      name: inflation_layer
      type: "costmap_2d::InflationLayer"

